pkg/kubelet/kubelet.go

```diff
@@ -1423,7 +1423,7 @@ func (kl *Kubelet) GetKubeletContainerLogs(podFullName, containerName, tail stri
 			return fmt.Errorf("failed to get status for pod %q - %v", podFullName, err)
 		}
 	}
-	if podStatus.Phase != api.PodRunning {
+	if podStatus.Phase != api.PodRunning || podStatus.Phase != api.PodSucceeded || podStatus.Phase != api.PodFailed {
 		return fmt.Errorf("pod %q is not in 'Running' state - State: %q", podFullName, podStatus.Phase)
```

Comment:
Fix the error message as well?

Wondering if a "case" statement would make this more readable, possibly/probably not...